src/bindings/generator/src/dbus_typing.py

```diff
@@ -0,0 +1,148 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+from typing import Tuple
+import dasbus.typing as DBusTypes
+
+basic_types = ["y", "b", "n", "q", "i", "u", "x", "t", "d", "s", "o", "g", "h", "v"]
+
+
+def map_basic_type_to_pytype(t: str) -> str:
+    if t == "y":
```

Comment:
The soup letters seems a good candidate for macros/constants as DBusTypes did.

Reply:
Good point. I refactored the `dbus_typing` to use the dasbus DBusTypes. Since its provided as dict that maps `DBusType -> str` I used that and reversed it to have `str -> DBusType`.